.github/workflows/ci.yml

```diff
@@ -67,7 +67,7 @@ jobs:
         with:
           tool: cargo-hack
       - name: Tests
-        run: cargo hack test --feature-powerset --depth 2 --clean-per-run --partition ${{ matrix.partition }}
+        run: cargo hack test --feature-powerset --exclude-features device-requests --depth 2 --clean-per-run --partition ${{ matrix.partition }}
```

Comment:
Do we really need exclusion? It would affect other developers who runs tests locally / in cloud

Instead, we could consider not failing such tests, but run only when it's possible. And would be better to print a warning if we can't run the test